src/platform/testing/unit/schemaform-utils.jsx

```diff
@@ -74,19 +75,33 @@ export class DefinitionTester extends React.Component {
 
   handleChange = data => {
     const { schema, uiSchema, formData } = this.state;
-    const { pagePerItemIndex, arrayPath } = this.props;
+    const { pagePerItemIndex, arrayPath, updateFormData } = this.props;
 
     let fullData = data;
 
     if (arrayPath) {
       fullData = set([arrayPath, pagePerItemIndex], data, formData);
     }
 
-    const { data: newData, schema: newSchema } = updateSchemaAndData(
-      schema,
-      uiSchema,
-      fullData,
-    );
+    const newSchemaAndData = updateSchemaAndData(schema, uiSchema, fullData);
+
+    let newData = newSchemaAndData.data;
+    const newSchema = newSchemaAndData.schema;
+
+    if (typeof updateFormData === 'function') {
+      if (arrayPath && typeof pagePerItemIndex === 'undefined') {
+        // Adding this console message to help with troubleshooting
+        // eslint-disable-next-line no-console
+        console.error(
+          'pagePerItemIndex prop is required when arrayPath is specified',
+        );
```

Comment:
Troubleshooting this source of a React error was very time consuming. So that's why Jerek & I discussed adding this console message here.